src/lowlevel.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2021 The Dogecoin Core developers
```

Comment:
Right now, the functionality defined by this header file is only used in `random.cpp`, what is the benefit of creating a new header file over using `random.h`?

Reply:
I expect changes that I will be making in the future to use low level functionality including intrinsics, assembly and other fancy things. I'd rather have one place for low level stuff, it just so happens that first place where I implemented such functionalities is related to random.h(.cpp).